docs/sdk_performance.md

```diff
@@ -1,5 +1,7 @@
 # SDK Performance
 
+**Note**: This document is outdated, it will be updated later.
```

Comment:
This line isn't appropriate for our public docs. If you don't have time to update now, can you remove the out of date parts, and perhaps replace them with a summary? It's better to have incomplete docs than incorrect ones.